openapi/generated/errors.ts

```diff
@@ -0,0 +1,49 @@
+
+/**
+   smart_events error codes
+
+    ```ts
+    apiCall.then((data) => {
+        console.log(data?.data.items)
+    }).catch((err) => {
+      if(APIErrorCodes.ERROR_5 == err.response?.data.code) {
+        // Handle error
+      }
+    })
+    ```
+*/
+export const APIErrorCodes = {
+  /** You tried to create an object which already exist in our repository*/
+  ERROR_1 : "OPENBRIDGE-1", 
+
+  /** You tried a life cycle transition which is not allowed*/
+  ERROR_2 : "OPENBRIDGE-2", 
+
+  /** The requested item does not exist in out repository*/
```

Comment:
@kelvah `out` or `our` repository?

Reply:
@sjamboro this file has been generated from the `API`
![image](https://user-images.githubusercontent.com/22591802/170940486-6ee56f2b-b1e8-44b6-a3ef-770269f578c8.png)
As soon as it will be fixed on that side, it will be possible to generate `errors.ts` again.
So for now, what is in place in this file looks correct to me.